.github/workflows/linkcheck.yml

```diff
@@ -19,11 +19,10 @@ jobs:
 
       # accept 429 only for PR runs where secret isn't available
       - name: Check links
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
         run: |
-          nix run nixpkgs#lychee -- \
+          nix develop .#linkcheck -c lychee \
             --verbose \
             --cache \
-            --accept '${{ github.event_name == 'pull_request' && '200, 429' || '200' }}' \
+            --cache-exclude-status "429" \
```

Comment:
That's a great flag. I was looking into a different approach that would cleanup the `.lycheecache` file but this is much cleaner :+1: